pubspec.yaml

```diff
@@ -4,27 +4,34 @@ description: Checks for missing, under-promoted, over-promoted, and unused depen
 homepage: https://github.com/Workiva/dependency_validator
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
+
+dependency_overrides:
+  build_config:
```

Comment:
what's the story with the override?

Reply:
Just waiting for the NNBD release of it - it's been merged, but not published:
https://github.com/dart-lang/build/blob/master/build_config/CHANGELOG.md#100-dev

I was keeping an eye on the repo and they've still go a couple of open NNBD PRs so I figured it would get published once those merge.